natura-cosmeticos/natds-rn-samples
# android integration

## Summary

This adds a consumable Android module created from `SampleTS` that contains a page showing all icons from the version `0.14` of **natds-icons**.

### Proposed changes

This PR brings the following changes:

- Add...;
- Change 2;
- Change 3.

### Related issue

This is part of the [DSY-1543 Criar segundo sample RN para integração ANDROID](https://natura.atlassian.net/browse/DSY-1543) Jira issue

### Dependencies added/removed (if applicable)

- Add: `@naturacosmeticos/natds-icons`: "0.14.0"
- Add: `@naturacosmeticos/natds-rn`: "^3.0.0"
- Add: `react-dom`: "^16.13.1"
- Add: `styled-components`: "5.1.0"

---

### Testing

I did not add or update any automated tests.

#### How to test

To verify the changes included in this, you may want to:

1. Go to `sampleTS/`
2. Run `yarn install` and `npx react-native link`
3. Run `yarn android:build:bundle` and `yarn android:generate:aar`
3. Check if the file `rnsamplefixed-release.aar` was created in `sampleTS/android/rnsamplefixed/build/outputs/aar/rnsamplefixed-release.aar`

**The file should be consumable inside a properly configured Android application.**

#### Test configuration

- Node.js version (if applicable): v14.8.0
- NPM version (if applicable): 6.14.7
- Yarn version (if applicable): 1.22.4

---

## Checklist
